basxconnect/mailer_integration/abstract/abstract_mailer_person.py

```diff
@@ -26,3 +26,19 @@ def interests_ids(self) -> typing.List[str]:
     @abc.abstractmethod
     def status(self) -> str:
         pass
+
```

Comment:
I will remove this unnecessary abstract class in the next PR ;)

Reply:
I am always in favour of removing code :D

Reply:
I merged my other work into this PR already, so that I don't have to juggle between branches..